### PR TITLE
feat: add dropdown with states of components and development zone

### PR DESCRIPTION
**Pull request by wilsson**

### CODE REVIEW

File: examples/basic/src/Button/Button.jsx [hunk 1]
```diff
@@ -12,27 +14,60 @@ const ButtonStyled = styled.button`
...
-export const Button = ({ children, outline }) => (
-  <ButtonStyled outline={outline}>{children}</ButtonStyled>
+export const Button = ({ children, outline, variant }) => (
+  <ButtonStyled variant={variant} outline={outline}>{children}</ButtonStyled>
 );
 
 Button.propTypes = {
   outline: p.bool,
   disabled: p.bool,
   /** My description for loading prop */
```
Comment (juanpablocs) [inline examples/basic/src/Button/Button.jsx:70]: innecesary comment

File: packages/papyrum-cli/src/components/Root.tsx [hunk 2]
```diff
@@ -49,91 +39,32 @@ const providerComponents = {
...
-      </IconWrapper> 
-    </RigthWrapper>
-    <LeftWrapper>
-      <Label>Ready</Label>
-    </LeftWrapper>
-  </TabWrapper>
-);
+const getMetadata = (stateForComponent: Object, stateSelected: string) => {
+  const { pathname } = location;
+  const [ state ] = stateForComponent[pathname].filter((e) => {
```
Comment (juanpablocs) [inline packages/papyrum-cli/src/components/Root.tsx]: why not?
```js
const [ state ] = stateForComponent[pathname].filter(({ name }) => name === stateSelected);
```

File: packages/papyrum-ui-docs/src/components/Playground/Playground.tsx [hunk 3]
```diff
@@ -1,21 +1,45 @@
...
+  useEffect(() => {
+    if(!stateForComponent[pathname]) {
+      stateForComponent[pathname] = [];
+      stateForComponent[pathname].push({
+        name: name,
+        code,
+        scope
+      });
+      setStateForComponent({ ...stateForComponent })
+    } else {
```
Comment (juanpablocs) [inline packages/papyrum-ui-docs/src/components/Playground/Playground.tsx:31]: `else` is the devil :smiling_imp: 

refactorize the block code

File: packages/papyrum-ui-docs/src/components/Playground/Playground.tsx [hunk 4]
```diff
@@ -1,21 +1,45 @@
...
+      !stateForComponent[pathname]
```
Comment (juanpablocs) [inline packages/papyrum-ui-docs/src/components/Playground/Playground.tsx:32]: `![].map()` ?

imminent error. your with sleep? :sleeping:

File: packages/papyrum-ui-docs/src/components/Playground/Playground.tsx [hunk 5]
```diff
@@ -1,21 +1,45 @@
...
+        .map(state => state.name)
+        .includes(name) && stateForComponent[pathname].push({
+          name: name,
+          code,
+          scope
+        });
+      //!stateForComponent[pathname].includes(name) && stateForComponent[pathname].push({name: code});
```
Comment (juanpablocs) [inline packages/papyrum-ui-docs/src/components/Playground/Playground.tsx]: remove comment

File: packages/papyrum-ui/src/components/Tab/Tab.tsx [hunk 6]
```diff
@@ -1,84 +1,24 @@
...
-          </LinkAction>
-          <Separate />
-          <LinkAction>
-            <Maximize size={20} />
-          </LinkAction>
-        </div>
-      </WrapperActions>}
+     {children.map((child) => {
+       const { name } = child.props;
+       return(
```
Comment (juanpablocs) [inline packages/papyrum-ui/src/components/Tab/Tab.tsx]: what is purpose for this?

File: packages/papyrum-ui/src/components/ToolBar/ToolBar.tsx [hunk 7]
```diff
@@ -1,10 +1,62 @@
...
+        >
+          Development
+        </TabItem>
+        {activePanel === 'development' && (
+          <>
+            <Separator />
+            <IconWrapper onClick={(e) => setShow(!show)}>
+              <Layers size={15} color="#5B5B5B" />
+              {show && (
+                <Dropdown list={stateForComponent as any} onClick={(state) => setStateSelected(state) }/>
```
Comment (juanpablocs) [inline packages/papyrum-ui/src/components/ToolBar/ToolBar.tsx]: `stateForComponent as any` ?

Comment (juanpablocs) [reply packages/papyrum-ui/src/components/ToolBar/ToolBar.tsx]: create your `interface` please :cry:

**Review by juanpablocs — CHANGES_REQUESTED** [review]
please resolve reviews ;)

File: examples/basic/src/Button/Button.jsx (hunk 1 above)
Comment (wilsson) [reply examples/basic/src/Button/Button.jsx:70]: That comment is for the basic example and demonstrates the use of the Props component.

File: packages/papyrum-ui-docs/src/components/Playground/Playground.tsx (hunk 5 above)
Comment (wilsson) [reply packages/papyrum-ui-docs/src/components/Playground/Playground.tsx]: That shouldn't be, thanks

File: examples/basic/src/Button/Button.jsx (hunk 1 above)
Comment (juanpablocs) [reply examples/basic/src/Button/Button.jsx:70]: The "tio Bob" is against this comment :smile:

File: packages/papyrum-ui/src/components/Tab/Tab.tsx (hunk 6 above)
Comment (wilsson) [reply packages/papyrum-ui/src/components/Tab/Tab.tsx]: that should not exist

**Comment by juanpablocs** [comment]
#7 resolve this

**Pull request merged**